### PR TITLE
docs: Quick Javadoc fixes.

### PR DESCRIPTION
**Pull request by GPopZach**
Added @code tag wherever < or > were used, also removed a few
self-closing <p> tags. Both of these issues cause errors with the latest
version of Javadoc.

Signed-off-by: Zachary Trant <zach@graalonline.com>

### CODE REVIEW

File: src/main/java/io/reactivex/rxjava3/internal/fuseable/FuseToFlowable.java [hunk 1]
```diff
@@ -21,8 +21,8 @@
  * the operator goes from Flowable to some other reactive type and then the sequence calls
  * for toFlowable again:
  * <pre>
- * Single&lt;Integer> single = Flowable.range(1, 10).reduce((a, b) -> a + b);
- * Flowable&lt;Integer> flowable = single.toFlowable();
+ * {@code Single<Integer> single = Flowable.range(1, 10).reduce((a, b) -> a + b);}
+ * {@code Flowable<Integer> flowable = single.toFlowable();}
```
Comment (akarnokd) [inline src/main/java/io/reactivex/rxjava3/internal/fuseable/FuseToFlowable.java]: These should be part of the same code block

File: src/main/java/io/reactivex/rxjava3/internal/fuseable/FuseToMaybe.java [hunk 2]
```diff
@@ -21,8 +21,8 @@
  * the operator goes from Maybe to some other reactive type and then the sequence calls
  * for toMaybe again:
  * <pre>
- * Single&lt;Integer> single = Maybe.just(1).isEmpty();
- * Maybe&lt;Integer> maybe = single.toMaybe();
+ * {@code Single<Integer> single = Maybe.just(1).isEmpty();}
+ * {@code Maybe<Integer> maybe = single.toMaybe();}
```
Comment (akarnokd) [inline src/main/java/io/reactivex/rxjava3/internal/fuseable/FuseToMaybe.java]: These should be part of the same code block.

File: src/main/java/io/reactivex/rxjava3/internal/fuseable/FuseToObservable.java [hunk 3]
```diff
@@ -21,8 +21,8 @@
  * the operator goes from Observable to some other reactive type and then the sequence calls
  * for toObservable again:
  * <pre>
- * Single&lt;Integer> single = Observable.range(1, 10).reduce((a, b) -> a + b);
- * Observable&lt;Integer> observable = single.toObservable();
+ * {@code Single<Integer> single = Observable.range(1, 10).reduce((a, b) -> a + b);}
+ * {@code Observable<Integer> observable = single.toObservable();}
```
Comment (akarnokd) [inline src/main/java/io/reactivex/rxjava3/internal/fuseable/FuseToObservable.java]: These should be part of the same code block.

File: src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java [hunk 4]
```diff
@@ -53,7 +53,7 @@
  * thread pool:
  *
  * <pre>
- * Scheduler limitScheduler = Schedulers.computation().when(workers -> {
+ * {@code Scheduler limitScheduler = Schedulers.computation().when(workers ->} {
```
Comment (akarnokd) [inline src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java]: This breaks the formatting of the lambda body.

File: src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java [hunk 5]
```diff
@@ -71,7 +71,7 @@
  * to the second.
  *
  * <pre>
- * Scheduler limitScheduler = Schedulers.computation().when(workers -> {
+ * {@code Scheduler limitScheduler = Schedulers.computation().when(workers ->} {
```
Comment (akarnokd) [inline src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java]: This breaks the formatting of the lambda body.

File: src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java [hunk 6]
```diff
@@ -84,9 +84,9 @@
  * algorithm).
  *
  * <pre>
- * Scheduler slowScheduler = Schedulers.computation().when(workers -> {
+ * {@code Scheduler slowScheduler = Schedulers.computation().when(workers ->} {
```
Comment (akarnokd) [inline src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java]: This breaks the formatting of the lambda body.

File: src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java [hunk 7]
```diff
@@ -84,9 +84,9 @@
...
  *  // use concatenate to make each worker happen one at a time.
- *  return Completable.concat(workers.map(actions -> {
+ *  {@code return Completable.concat(workers.map(actions ->} {
```
Comment (akarnokd) [inline src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java]: This breaks the formatting of the lambda body.

File: src/test/java/io/reactivex/rxjava3/internal/operators/flowable/BlockingFlowableNextTest.java [hunk 8]
```diff
@@ -223,8 +223,8 @@ public void nextWithCallingHasNextMultipleTimes() {
 
     /**
      * Confirm that no buffering or blocking of the Observable onNext calls occurs and it just grabs the next emitted value.
-     * <p/>
-     * This results in output such as => a: 1 b: 2 c: 89
+     * <p>
+     * {@code This results in output such as => a: 1 b: 2 c: 89}
```
Comment (akarnokd) [inline src/test/java/io/reactivex/rxjava3/internal/operators/flowable/BlockingFlowableNextTest.java]: No reason to include "This results in output such as" in the code block.

File: src/test/java/io/reactivex/rxjava3/internal/operators/observable/BlockingObservableNextTest.java [hunk 9]
```diff
@@ -227,8 +227,8 @@ public void nextWithCallingHasNextMultipleTimes() {
 
     /**
      * Confirm that no buffering or blocking of the Observable onNext calls occurs and it just grabs the next emitted value.
-     * <p/>
-     * This results in output such as => a: 1 b: 2 c: 89
+     * <p>
+     * {@code This results in output such as => a: 1 b: 2 c: 89}
```
Comment (akarnokd) [inline src/test/java/io/reactivex/rxjava3/internal/operators/observable/BlockingObservableNextTest.java]: No reason to include "This results in output such as" in the code block.

File: src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java [hunk 10]
```diff
@@ -340,11 +340,11 @@ public void onNext(String t) {
      *
      * When using SynchronizedSubscriber we get this output:
      *
-     * p1: 18 p2: 68 => should be close to each other unless we have thread starvation
+     * {@code p1: 18 p2: 68 => should be close to each other unless we have thread starvation}
```
Comment (akarnokd) [inline src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java]: No reason to include "should be close to each other unless we have thread starvation" in the code block.

File: src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java [hunk 11]
```diff
@@ -340,11 +340,11 @@ public void onNext(String t) {
...
      *
      * When using SerializedObserver we get:
      *
-     * p1: 1 p2: 2445261 => should be close to each other unless we have thread starvation
+     * {@code p1: 1 p2: 2445261 => should be close to each other unless we have thread starvation}
```
Comment (akarnokd) [inline src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java]: No reason to include "should be close to each other unless we have thread starvation" in the code block.

File: src/test/java/io/reactivex/rxjava3/subscribers/SerializedSubscriberTest.java [hunk 12]
```diff
@@ -341,11 +341,11 @@ public void onNext(String t) {
      *
      * When using SynchronizedSubscriber we get this output:
      *
-     * p1: 18 p2: 68 => should be close to each other unless we have thread starvation
+     * {@code p1: 18 p2: 68 => should be close to each other unless we have thread starvation}
```
Comment (akarnokd) [inline src/test/java/io/reactivex/rxjava3/subscribers/SerializedSubscriberTest.java]: No reason to include "should be close to each other unless we have thread starvation" in the code block.

File: src/test/java/io/reactivex/rxjava3/subscribers/SerializedSubscriberTest.java [hunk 13]
```diff
@@ -341,11 +341,11 @@ public void onNext(String t) {
...
      *
      * When using SerializedSubscriber we get:
      *
-     * p1: 1 p2: 2445261 => should be close to each other unless we have thread starvation
+     * {@code p1: 1 p2: 2445261 => should be close to each other unless we have thread starvation}
```
Comment (akarnokd) [inline src/test/java/io/reactivex/rxjava3/subscribers/SerializedSubscriberTest.java]: No reason to include "should be close to each other unless we have thread starvation" in the code block.

File: src/test/java/io/reactivex/rxjava3/tck/BaseTck.java [hunk 14]
```diff
@@ -51,7 +51,7 @@ public long maxElementsFromPublisher() {
 
     /**
      * Creates an Iterable with the specified number of elements or an infinite one if
-     * elements > {@link Integer#MAX_VALUE}.
+     * {@code elements > {@link Integer#MAX_VALUE}.}
```
Comment (akarnokd) [inline src/test/java/io/reactivex/rxjava3/tck/BaseTck.java]: This breaks the link I think.

**Review by akarnokd — CHANGES_REQUESTED** [review]
These are internal or test files which are exempt from the JavaDocs generation.

File: src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java (hunk 10 above)
Comment (GPopZach) [reply src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java]: You're correct, just did the whole line for cosmetic reasons, but I can surely remove the rest of the statement from it.

File: src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java [hunk 15]
```diff
@@ -84,13 +88,15 @@
  * algorithm).
  *
  * <pre>
+ * {@code 
  * Scheduler slowScheduler = Schedulers.computation().when(workers -> {
  *  // use concatenate to make each worker happen one at a time.
- *  return Completable.concat(workers.map(actions -> {
+ *  return Completable.concat(workers.map(actions ->} {
```
Comment (akarnokd) [inline src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java]: There is an unnecessary `}` here.

File: src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java [hunk 16]
```diff
@@ -1043,7 +1043,7 @@ public void run() {
             };
 
             TestHelper.race(r1, r2);
-
+
```
Comment (akarnokd) [inline src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java]: Please remove these extra spaces.

**Review by GPopZach — COMMENTED** [review]
Took all the comments from @akarnokd into account and fixed it up,

File: src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java (hunk 15 above)
Comment (GPopZach) [reply src/main/java/io/reactivex/rxjava3/internal/schedulers/SchedulerWhen.java]: RIP

**Review by GPopZach — COMMENTED** [review]
Fixed.

File: src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java (hunk 16 above)
Comment (GPopZach) [reply src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java]: > Please remove these extra spaces.

Done.

**Review by akarnokd — APPROVED** [review]

**Pull request merged**